examples/bri-3/src/bri/transactions/capabilities/getAllTransactions/getAllTransactionsQuery.handler.ts

```diff
@@ -0,0 +1,24 @@
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+import { TransactionDto } from '../../api/dtos/response/transaction.dto';
+import { TransactionStorageAgent } from '../../agents/transactionStorage.agent';
+import { GetAllTransactionsQuery } from './getAllTransactions.query';
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import { Transaction } from '../../models/transaction';
+
+@QueryHandler(GetAllTransactionsQuery)
+export class GetAllTransactionsQueryHandler
+  implements IQueryHandler<GetAllTransactionsQuery>
+{
+  constructor(
+    @InjectMapper() private readonly mapper: Mapper,
```

Comment:
usually you inject without readonly, please make it consistent

Reply:
let me check which all places are without readonly. I'll make the change